src/services/email.service.js

```diff
@@ -0,0 +1,42 @@
+import { config } from 'dotenv';
```

Comment:
Can't remember us agreeing to use this format for our file names `email.service`. If you are going to use this, then you'll have to change all non-generated files to the same format. Our codebase has to be consistent